mailbox/backup/src/main/java/org/apache/james/mailbox/backup/zip/package-info.java

```diff
@@ -0,0 +1,41 @@
+/****************************************************************
+ * Licensed to the Apache Software Foundation (ASF) under one   *
+ * or more contributor license agreements.  See the NOTICE file *
+ * distributed with this work for additional information        *
+ * regarding copyright ownership.  The ASF licenses this file   *
+ * to you under the Apache License, Version 2.0 (the            *
+ * "License"); you may not use this file except in compliance   *
+ * with the License.  You may obtain a copy of the License at   *
+ *                                                              *
+ *   http://www.apache.org/licenses/LICENSE-2.0                 *
+ *                                                              *
+ * Unless required by applicable law or agreed to in writing,   *
+ * software distributed under the License is distributed on an  *
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY       *
+ * KIND, either express or implied.  See the License for the    *
+ * specific language governing permissions and limitations      *
+ * under the License.                                           *
+ ****************************************************************/
+
+/**
+ * The zip archive format for the backup of an account
+ * This archive contains the following entries for each mailbox of the user:
+ *<ul>
+ * <li>'mailboxName/' : directory entry</li>
+ * <li>'mailboxName/annotations/' : directory entry, optional. Is present only if the mailbox contains some annotations</li>
+ * <li>'mailboxName/annotations/annotation1Key' : file entry which name is the key of the annotation,
+ *                                              the content of the annotation is stored in the content of the file</li>
+ * <li>'mailboxName/annotations/annotation2Key' : idem a mailbox may contains several annotations</li>
+ * <li>'messageId1' : file entry which name is the serialized {@link org.apache.james.mailbox.model.MessageId} of the message,
+ *                  the raw RFC822 message is stored in the content of the file</li>
+ * <li>'messageId2' : a mailbox may contains several messages.</li>
```

Comment:
s/a mailbox/an account/?

Reply:
No, it's the list of messages for the mailbox entry.

Reply:
messages are not in mailbox, but flat in the archive

Reply:
Yes, but the messages of a mailbox are inserted just after the mailbox and its annotations in the zip file.  
Maybe the term contains is not the best. 
Maybe : 'A mailbox is followed by all its messages.'

I tought that the last paragraph explained well enough the iterative structure of the file. 
But apparently it's not clear enougth.

What would you change to emphasis the fact that the insertion order is important and must be like that : 

- **mailbox1**
- mailbox1/annotations/
- /annotations/annotation1Key
- message1ForMailbox1
- message2ForMailbox1
- **mailbox2**
- ...

Reply:
oh thanks for the explanation, I was not aware of that. Yes, it should be clearer like that.